src/main/java/org/gridsuite/loadflow/server/dto/ParameterInfos.java

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright (c) 2022, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.dto;
+
+import com.powsybl.commons.parameters.ParameterType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import java.util.List;
+
+/**
+ * @author David Braquart <david.braquart@rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@Builder
+public class ParameterInfos {
```

Comment:
com.powsybl.commons.parameters.Parameter is already serializable (adding the correct jackson module) and could be used instead